entries/d/deepl.com.json

```
@@ -0,0 +1,13 @@
+{
+  "DeepL": {
+    "domain": "deepl.com",
+    "contact": {
+      "facebook": "DeepLcom",
+      "twitter": "DeepLcom"
+    },
+    "keywords": [
+      "developer",
+      "utilities"
```

```suggestion
            "developer"
```
The Utilities category is meant for services like internet providers, phone carriers, gas and electricity companies.